### DISCUSSION TITLE
Laravel Breeze Flux

### DISCUSSION THREAD

**Discussion by RiadhKHEDHIRI** [livewire/flux#353] [Feature Requests]
Hi,

I would like to have Laravel Breeze with Flux.

Thanks!

**Comment by jeppeolesen** [comment]
The Breeze views are published in the Laravel projects, and it's relatively painless and quick to refactor them to flux yourself.

With Flux being a paid package and Breeze also offering Vue or React scaffolds, adding it as a standard option for new installs is unlikely.
